nf_core/subworkflow-template/tests/test.yml

```diff
@@ -1,7 +1,7 @@
 ## TODO nf-core: Please run the following command to build this file:
 #                nf-core subworkflows create-test-yml {{ subworkflow_name_underscore }}
 - name: "{{ subworkflow_name }}"
-  command: nextflow run ./tests/subworkflows/nf-core/{{ subworkflow_dir }} -entry test_{{ subworkflow_name }} -c ./tests/config/nextflow.config
+  command: nextflow run ./tests/subworkflows/{{ org }}/{{ subworkflow_dir }} -entry test_{{ subworkflow_name }} -c ./tests/config/nextflow.config
```

Comment:
Why do we not pass the `nextflow.config` that sits alongside the test `main.nf` explicitly here as we do for `modules`?

Reply:
It's not required, we still have it in modules to avoid changing too many files (see [#1562]( https://github.com/nf-core/tools/issues/1562)), but now that we are starting fresh with subworkflows we don't need to add it.

Reply:
What exactly isn't required? We still create a local config file in `tests/subworkflows/<org>/<TOOL>`

Reply:
Sorry I didn't explain good 😅 We don't need to pass the path of this file explicitly, as it's located in the same folder as `main.nf`, it's taken into account by Nextflow by default.